src/i18n/locales/en-US.json

```diff
@@ -177,6 +177,12 @@
   "components.settings.applicationsettingsscreen.termsOfUse": "Terms of Use",
   "components.settings.applicationsettingsscreen.title": "Settings",
   "components.settings.applicationsettingsscreen.appSettingsTitle": "App settings",
+  "components.settings.applicationsettingsscreen.version": "Current Version",
+  "components.settings.applicationsettingsscreen.commit": "Commit",
+  "components.settings.applicationsettingsscreen.walletType": "Wallet type",
+  "components.settings.applicationsettingsscreen.byronWallet": "Byron-era wallet",
+  "components.settings.applicationsettingsscreen.shelleyWallet": "Shelley-era wallet",
+  "components.settings.applicationsettingsscreen.unknownWalletType": "Unknown Wallet Type",
   "components.settings.biometricslinkscreen.enableFingerprintsMessage": "Enable use of fingerprints in device settings first!",
```

Comment:
I think most of these keys are in `.walletsettingscreen.`

Reply:
It is with that id in develop branch: https://github.com/Emurgo/yoroi-mobile/blob/develop/src/i18n/locales/en-US.json#L165

Its restored in this branch because It had been removed in `[YOMO-166-app-settings-bis](https://github.com/Emurgo/yoroi-mobile/tree/YOMO-166-app-settings-bis)` branch

Reply:
In the revamp, commit text is `Commit:` but in the revamp is `:`